project/cmake/installdata/common/common.txt

```diff
@@ -3,11 +3,14 @@ sounds/*
 system/keymaps/*
 system/library/*
 system/players/VideoPlayer/etc/*
+system/python/*
 system/shaders/*
 system/settings/*
 userdata/*
 system/addon-manifest.xml
 system/colors.xml
+system/IRSSmap.xml
 system/peripherals.xml
 system/playercorefactory.xml
+system/X10-Lola-IRSSmap.xml
```

Comment:
I didn't put them in windows only just for the heck of it. They are only
used on windows, that is, they are the same file, the lola one is an
example for a specific remote and wont be used by the installation as such.
This is keymaps for 'lirc for windows'.
7. apr. 2016 15:43 skrev "Cristiano A. Silva" <notifications@github.com>:

> In project/cmake/installdata/common/common.txt
> <https://github.com/xbmc/xbmc/pull/9568#discussion_r58872648>:
>
> >  system/peripherals.xml
> >  system/playercorefactory.xml
> > +system/X10-Lola-IRSSmap.xml
>
> I don't know if they are used at all under linux. I just want that diff
> -qr autotools/ cmake/ | sort returns a nice *nada*. [image: :wink:]
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/9568/files/5b9d2d52b4166c3e2e00be21ebad70f596d9b4c6..d8f6977c16c01c2a53242814a9c7494e8e93eb99#r58872648>
>